manim/mobject/types/vectorized_mobject.py

```diff
@@ -1096,6 +1260,134 @@ def get_subpaths(self) -> tuple:
         """
         return self.get_subpaths_from_points(self.points)
 
+    def get_subpath_split_indices_from_points(
+        self, points: np.ndarray, n_dims: int = 3, strip_null_end_curves: bool = False
+    ) -> np.ndarray:
+        nppcc = self.n_points_per_cubic_curve
+        starts = points[::nppcc]
+        ends = points[nppcc - 1 :: nppcc]
+        # This ensures that there are no more starts than ends.
+        # TODO: ends.shape[0] would be more efficient, but some test cases regarding
+        # Flash and ShowPassing flash expect a Python list instead of an ndarray, so
+        # ends.shape[0] breaks those test cases. Fix these inconsistencies.
+        n_curves = len(ends)
+        starts = starts[:n_curves]
+
+        # DO NOT DELETE THIS! Or else, a [[0 0]] ndarray will be returned
+        # for an empty list of points, when it should instead return [].
```

Comment:
The latter part of the comment suggests that an empty Python list is returned, which is not the case. The wording should be improved.

Reply:
I see. I'm going to check this as well.

Reply:
Done!